packages/theming/theming-utils/src/mapPipelineToTheme.android.ts

```diff
@@ -75,7 +77,10 @@ export function mapPipelineToTheme(pipelineOutput: any): AliasColorTokens {
     neutralStrokeDisabled: pipelineOutput.neutralStrokeDisabled.strokeColorRest,
 
     //Brand tokens.
-    brandBackground: pipelineOutput.brandBackground1.fillColorRest,
+    brandBackground: 'green',
+
+    // brandBackground: pipelineOutput.brandBackground1.fillColorRest,
```

Comment:
This is just indicating the colors I overrode to easy check that the tester react native controls pick the overrides correctly for the theme property being accessed. The 3 updated values here are the ones referred in styles for the react native components in the test sample code.